packages/vue/src/utils/deep-to-raw.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  isProxy,
+  isReactive,
+  isRef,
+  toRaw,
+} from 'vue'
+
+export function deepToRaw<T>(input: T): T {
+  if (Array.isArray(input)) {
+    return [...input.map(deepToRaw) as unknown[]] as T
+  }
+
+  if (isRef(input)
+    || isReactive(input)
+    || isProxy(input)) {
+    return deepToRaw(toRaw(input))
+  }
+
+  if (input !== null && typeof input === 'object') {
+    const clone = {}
```

Comment:
```suggestion
    const clone = {}
```

Emmmm... I remember that using `{}` directly causes issues... let's ignore this (current one) comment now but leave it as a comment / note for later reference.

Reply:
The AI SDK copied in part of the code, the general data is plain object, so it shouldn't be too much of a problem